package.json

```diff
@@ -83,6 +85,6 @@
     "typescript": "^4.5.4"
   },
   "engines": {
-    "node": ">=16.0"
+    "node": "^14.0 || >=16.0"
```

Comment:
Why not just `>=14.0`?
Would this also work in Node.js 12? Because if it did then we should make this `>= 12.0` to support the widest range possible.